modules/restore-deployment-config-helm.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/restore-acs.adoc
+:_module-type: PROCEDURE
+[id="restore-deployment-config-helm_{context}"]
+= Restore Central deployment using Helm
+
+[role="_abstract"]
+You can restore your Central deployment to its original configuration by using Helm.
+To successfully restore, you need the backup of your Central custom resource, `central-tls`, and the administrator password.
+
+.Prerequisites
+* You must have the Helm values backup file.
+* You must have installed the `roxctl` CLI.
```

Comment:
You must have the database backup file in order to generate `values-private.yaml`